HomeWork/src/main/java/arraylist/TasksWithList.java

```diff
@@ -1,20 +1,23 @@
 package arraylist;
 
+import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.Iterator;
 import java.util.List;
 import java.util.Objects;
 
-public class ArrayList {
+public class TasksWithList {
     public static void main(String[] args) {
         String[] str = {"A", "B", "C", "D"};
         Integer[] integers = {1, 3, 4, 5, 3, 21};
-        List<Integer> list = new java.util.ArrayList<>(Arrays.asList(1, 3, 2, 7, 3, 4, 4, 10, 4, 6, 6));
-        List<String> stringList = new java.util.ArrayList<>(Arrays.asList("fox", "list", "fox", "apple", "list", "cat",
+        List<Integer> list = new ArrayList<>(Arrays.asList(1, 3, 2, 7, 3, 4, 4, 10, 4, 6, 6));
+        List<String> stringList = new ArrayList<>(Arrays.asList("fox", "list", "fox", "apple", "list", "cat",
                 "orange", "orange", "dota", "Alex", "fox", "list", "apple", "blue"));
+
+        System.out.println(findUnique(list));
+
     }
 
-    static public List<String> findOccurrence(List<String> elements) {
+    public static List<String> findOccurrence(List<String> elements) {
```

Comment:
You should not return a list of strings. According to the task, you should return the following structure (which is not `List<String>`:
`[

   {name: "bird", occurrence: 2},

   {name: "fox", occurrence: 1},

   {name: "cat", occurrence: 1}

]`